### ISSUE TITLE
Constant and random crashing regardless of the situation

### ISSUE THREAD

**Issue by SentientCactus** [Iskallia/Vault-public#259]
**Describe the bug**
In basic terms I've found the modpack to be extremely unstable on my system regardless of what I'm running it from. If I run it form a launcher (MultiMC or CursedForge) it crashes in launching. Ram was set to 8gb even tested with 6 and 7 with the same results. Running the mod through the vanilla launcher seems to be the most stable situation in which you can launch the game as I've actually been able to play on the server that I own thought that although it will end up crashing eventually whether that be 5 mins or 3 hrs (it's more stable after the first 30 mins of playing). While it's still more stable it still ends up crashing at least 10 times before I even reach the title screen and that is if the modpack seemingly wants to launch that day. Take today for example I was playing the modpack perfectly fine the day before and I even got on and off multiple times during that day without issue, but when I come to try and launch it later today it just doesn't work any longer my thought are that the reason for this happening most likely has to do with certain discrete video cards as my friend doesn't have this problem, but I know others do. My friend is using integrated graphic (they can run it fine) and I'm using a discrete Radeon gpu (5600xt). That is my current guess, but I'm unable to understand the crash logs well enough to tell you what is actually wrong, which is why I'm attaching some of them. The reason why I can't read them as they are all hexdumps.

**To Reproduce**
1. Have my computer or another computer that is affected by it.
2. Download the modpack.
3. Run, if it works you won the roulette, but you may not be so lucky next time.

**Expected behavior**
Game crashes randomly.

**Screenshots or Crash Report**
If applicable, add screenshots or crash reports to help explain your problem.

**Info (please complete the following information):**
- World Type: [Yes]
- The_Vault Version [1.2] (side note 0.9 works fine for some reason)

**Additional context**
Fresh installs of the game don't fix it.
May be related to issue #235 as they had constant crashing, but not specific to launching the game and there was also no logs there.
If you'd like to reach me in another place that isn't here as I may not actively be viewing this because I don't commonly use github. Then my discord is SentientCactus#4134

(some error logs that I hope can be of help)
[hs_err_pid1416.log](https://github.com/Iskallia/Vault-public/files/6321277/hs_err_pid1416.log)
[hs_err_pid2716.log](https://github.com/Iskallia/Vault-public/files/6321278/hs_err_pid2716.log)
[hs_err_pid7628.log](https://github.com/Iskallia/Vault-public/files/6321279/hs_err_pid7628.log)
[hs_err_pid7656.log](https://github.com/Iskallia/Vault-public/files/6321280/hs_err_pid7656.log)
[hs_err_pid8704.log](https://github.com/Iskallia/Vault-public/files/6321281/hs_err_pid8704.log)
[hs_err_pid8720.log](https://github.com/Iskallia/Vault-public/files/6321282/hs_err_pid8720.log)
[hs_err_pid11896.log](https://github.com/Iskallia/Vault-public/files/6321283/hs_err_pid11896.log)
[hs_err_pid13128.log](https://github.com/Iskallia/Vault-public/files/6321284/hs_err_pid13128.log)
[hs_err_pid14196.log](https://github.com/Iskallia/Vault-public/files/6321285/hs_err_pid14196.log)
[hs_err_pid15096.log](https://github.com/Iskallia/Vault-public/files/6321286/hs_err_pid15096.log)
[hs_err_pid15432.log](https://github.com/Iskallia/Vault-public/files/6321287/hs_err_pid15432.log)
[hs_err_pid15576.log](https://github.com/Iskallia/Vault-public/files/6321288/hs_err_pid15576.log)
[hs_err_pid15852.log](https://github.com/Iskallia/Vault-public/files/6321289/hs_err_pid15852.log)
[hs_err_pid17384.log](https://github.com/Iskallia/Vault-public/files/6321290/hs_err_pid17384.log)

**Comment by magicalex234** [comment]
From the log that I opened, i believe the issue is that you ran out of memory. How many GB of ram do you have allocated? because the pack recommends 6-8

**Comment by SentientCactus** [comment]
Hey, I had 8 gb alloted, but about na hour ago, I did a fresh install of
java, so my guess is that is that my java courtupted when I updated it
between version 0.9 and 1.2. I figured that out about an hour ago. I also
tested it with older versions of java and the modpack doesn't like that, so
most people that report similar problems probably have a courtupted version
of java or thier java needs an update, bit thanks for trying to help.
